cmd/server.go

```diff
@@ -511,8 +505,7 @@ func init() {
 	serverCmd.AddCommand(serverHelpCmd)
 
 	// NOTE: sync these with forkCmd
-	serverCmd.Flags().String("consumer-prefix", "", "deprecated - use --consumer-suffix instead")
-	serverCmd.Flags().String("consumer-suffix", "", "a suffix to use for the consumer group name")
+	serverCmd.Flags().String("consumer-suffix", "", "suffix which is appended to the nats consumer group name")
```

Comment:
Honestly, I think we should drop the ability to suffix the consumer name entirely. Or at least make it hidden. If customers want to run multiple EDS servers, then that should be managed through the shopmonkey backend/HQ

Reply:
agreed but until we can get the import/restart working well i think we are going to need it for them. but agreed we shouldn't support this.

Reply:
As a customer who will be using two EDS servers in production in the not-so-distant future and who has set it up using separate consumer prefixes, how would this work?
> that should be managed through the shopmonkey backend/HQ

Reply:
@athomas-byrider we are building out EDS as a whole managed product now. You will soon be able to provision and manage your EDS servers in the UI, running still in your own infrastructure, if you still desire. Under the covers it will still use some sort of consumer naming to allow for multiple but that will be owned by us, since those consumers live in our nats space.

Reply:
@athomas-byrider we will be coming back to you in a few weeks to share all the new EDS v3 plans with you guys and a migration path but please don't move to this just yet!

Reply:
@jhaynie  We have an open ticket to upgrade our EDS version, so I'll make sure we pin it to the newest v2 at most.  Thanks for the info and look forward to hearing about v3 plans and future state